README.md

```diff
@@ -16,39 +16,27 @@ Mac users:
    ```
    psql -d cyf-bike-app-dev -f server/scripts/create_database.sql
    ```
+## Description
+We developed a tool to help a local charity, Bikes for Refugees, more efficiently schedule bike collections for their clients. Built using React (with Boostrap and vanilla CSS), Node.js and Express, and Postgresql for the database. This was our final project for the Code Your Future Full Stack Web Development course, of which we were in Scotland Class 5. We had four weeks to design and build our product and present it to an external audience. 
 
-# Starter Kit
+View it here: https://cyf-bike-app.herokuapp.com/
 
-[![Deploy to Heroku](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
 
-- [x] Full stack ES8+ with [Babel]
-- [x] [Node] LTS support (verified working on 12.x and 14.x LTS releases)
-- [x] [Express] server
-- [x] [React] client with [Webpack]
-- [x] Client-side routing with [React Router]
-- [x] Linting with [ESLint] and [Prettier]
-- [x] Dev mode (watch modes for client and server, proxy to avoid CORS issues)
-- [x] Production build (single deployment artifact, React loaded via CDN)
-- [x] [Heroku] deployment
-- [x] [Cloud Foundry] deployment
-- [x] [Docker] build
-- [x] [Postgres] database with [node-postgres]
+## Main Features 
+Main features include:
+- A waiting list page with user details and booking statuses, retrieved from a Postgresql database
+- Ability to select users from the list and send a customisable message
+- The message UUID is stored in the database
```

Comment:
```suggestion
```
We can probably remove this as it's not a feature you'd describe to users of the app. It's more of a code implementation detail